src/shared/components/case-viewer/case-event-trigger/case-event-trigger.component.ts

```diff
@@ -53,11 +53,11 @@ export class CaseEventTriggerComponent implements OnInit, OnDestroy {
     });
   }
 
-  ngOnDestroy() {
-    if (this.activityPollingService.isEnabled) {
+  public ngOnDestroy(): void {
+    if (this.activityPollingService && this.activityPollingService.isEnabled) {
```

Comment:
Wouldn't this suffice?
```suggestion
    if (this.activitySubscription) {
```